glide.yaml

```diff
@@ -94,3 +100,4 @@ import:
     version: v1.4.0
   - package: github.com/spf13/pflag
     version: v1.0.3
+  - package: github.com/boltdb/bolt
```

Comment:
It is better to describe the exact release version.
Currently, latest release is v1.3.1

Reply:
it is added.